### DISCUSSION TITLE
Self hosting: Can't upload photos from mobile app

### DISCUSSION THREAD

**Discussion by dnred** [ente-io/ente#2451] [Q&A] [answered]
I was trying out self hosting and uploading photos from the desktop app works but uploading photos from the mobile app doesn't. It says that it's encrypting backup and then it's seemingly done doing anything and next to photos I have a crossed cloud icon that indicates that it didn't sync.

**Comment by mnvr** [comment, marked as answer]
https://help.ente.io/self-hosting/troubleshooting/uploads
